example_configs/calibre_web.md

```diff
@@ -0,0 +1,95 @@
+# Configuration for Calibre-Web
+
+Replace `dc=example,dc=com` with your LLDAP configured domain.
+
+
+### Login type
+
+```
+Use LDAP Authentication
+```
+
+### LDAP Server Host Name or IP Address
+
+```
+192.168.X.X
```

Comment:
Maybe to be consistent with the rest, use "lldap" or "localhost"? Or offer both.